SimPEG/electromagnetics/analytics/__init__.py

```diff
@@ -1,6 +1,36 @@
 from .TDEM import hzAnalyticDipoleT, hzAnalyticCentLoopT
 from .FDEM import hzAnalyticDipoleF
-from .FDEMcasing import *
-from .DC import *
-from .FDEMDipolarfields import *
+from .FDEMcasing import (
+    getKc,
+    _r2,
```

Comment:
I know they get imported with the * command, but it's probably unnecessary to import the hidden methods here.

Reply:
I agree. I'll remove them and check if something fails.